tests/foreman/cli/test_syncplan.py

```diff
@@ -417,26 +412,27 @@ def test_positive_synchronize_custom_product_current_sync_date(self):
         :CaseLevel: System
         """
         interval = 60 * 60  # 'hourly' sync interval in seconds
+        delay = 20
+        product = make_product({'organization-id': self.org['id']})
+        repo = make_repository({'product-id': product['id']})
         sync_plan = self._make_sync_plan({
             'enabled': 'true',
             'interval': 'hourly',
             'organization-id': self.org['id'],
-            'sync-date': datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S"),
+            'sync-date': (
+              datetime.utcnow() - timedelta(interval - delay/2)
+            ).strftime("%Y-%m-%d %H:%M:%S"),
         })
-        product = make_product({'organization-id': self.org['id']})
-        repo = make_repository({'product-id': product['id']})
         # Associate sync plan with product
         Product.set_sync_plan({
             'id': product['id'],
             'sync-plan-id': sync_plan['id'],
         })
-        # Wait half of expected time
```

Comment:
Hello, I did the rename and I added some delay in front of the first assert. But the real problem here is that the way we check the repo sync doesn't catch sync in progress. But I'd like to fix this in a separate PR.